apps/sports-helsinki/src/domain/search/eventSearch/searchResultList/EventsOrderBySelect.tsx

```diff
@@ -0,0 +1,47 @@
+import { Select, useSearchTranslation } from 'events-helsinki-components';
+import { useRouter } from 'next/router';
+import React from 'react';
+import useHandleEventsOrderChange from '../../../../hooks/useHandleEventsOrderChange';
+import { DEFAULT_EVENT_SORT_OPTION, EVENT_SORT_OPTIONS } from '../constants';
+import styles from './eventsOrderBySelect.module.scss';
+
+const EventsOrderBySelect: React.FC = () => {
+  const { t } = useSearchTranslation();
+  const router = useRouter();
+  const sort = router.query?.sort ?? DEFAULT_EVENT_SORT_OPTION;
+  const handleEventsOrderChange = useHandleEventsOrderChange();
```

Comment:
Maybe this could simply be along the lines of:
```
useCallback(
     async (option: Option) => {
      router.push(
           {
             pathname: router.pathname,
             query: { ...router.query, ...{ sort: option.value } },
           },
     },
     [router.asPath, router.basePath, router.locale]
   );

```
if I understood correctly what is inside `useHandleEventsOrderChange` ..?

Reply:
Renamed and changed, see https://github.com/City-of-Helsinki/events-helsinki-monorepo/pull/174#discussion_r1088868662 for details